test/example/CMakeLists.txt

```diff
@@ -1,13 +1,19 @@
+find_package(GTest REQUIRED)
```

Comment:
It is required [here](https://github.com/BakcylProgramowania/paragon_back/blob/main/test/CMakeLists.txt) no need to repeat that